### PR TITLE
Add article development process

### PR DESCRIPTION
**Pull request by xfq**
[Preview](https://deploy-preview-590--i18n-drafts.netlify.app/pages/article-devt-process.html)

-----

based on https://www.w3.org/International/geo/2004/article-devt-process.html

### CODE REVIEW

File: pages/article-devt-process.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    draft and thought about it in advance.]</p>
+  <p>Comments on small editorial nits are likely to be less relevant at this stage. </p>
+</section>
+
+<section id="wd">
+  <h2>Stage: Working draft</h2>
+  <p class="begin"><strong>This stage begins when:</strong> <br>The author produces a new draft, incorporating all appropriate comments received on the
+    initial draft, and sends a call for comments on the WG mailing list.</p>
+  <p>The author should try to make this as near as possible to publication ready. See the
+    <a href="https://w3c.github.io/i18n-activity/guidelines/editing">editorial guidelines for i18n docs</a>.</p>
```
Comment (himorin) [inline pages/article-devt-process.html]: just a question, better to use w3.org URL or not? For this, https://www.w3.org/International/i18n-activity/guidelines/editing
or leave these as relative link but not absolute / full URL?

**Review by r12a — APPROVED** [review]
The list layout under Brainstorm looks a bit wierd.  Perhaps we can improve on that.  Otherwise, with the exception of Atsushi's comment, looks fine to me.

File: pages/article-devt-process.html (hunk 1 above)
Comment (xfq) [reply pages/article-devt-process.html]: Thanks. Should be fixed now.

**Comment by xfq** [comment]
I just found https://www.w3.org/International/i18n-activity/guidelines/editing#process

Perhaps we can just use it and close this PR?

**Comment by xfq** [comment]
Closing.

**Pull request closed without merging**